packages/harvester/harvester-cloud-provider/generated-changes/patch/Chart.yaml.patch

```diff
@@ -4,17 +4,17 @@
  annotations:
    catalog.cattle.io/certified: rancher
    catalog.cattle.io/display-name: Harvester Cloud Provider
--  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.24.0-0'
+-  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.26.0-0'
 +  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.25.0-0'
```

Comment:
Is this change expected?   
Rancher v2.7.2 will support k8s 1.25. We should drop this change if we want to show this chart in the downstream 1.25 cluster. 
I guess you might forget to drop it when adding this file back 😄